sklearn/calibration.py

```diff
@@ -80,15 +76,29 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
         If "prefit" is passed, it is assumed that base_estimator has been
         fitted already and all data is used for calibration.
 
-        .. versionchanged:: 0.22
-            ``cv`` default value if None changed from 3-fold to 5-fold.
+        .. versionchanged:: 0.20
+            ``cv`` default value if None will change from 3-fold to 5-fold
+            in v0.22.
+
+    ensemble : bool, optional
+        When ``cv`` is not "prefit", it determines how the final estimator
+        is fit.
+        If ``ensemble`` is True (default), an estimator (clone of
+        base_estimator) is fit and calibrated on each fold. The final
+        estimator is an ensemble that averages the predictions of all such
+        estimators.
+        When ``ensemble`` is False, the base_estimator is fit on the whole
+        ``X``, ``y`` data, and the cross validation generator is used to
+        get predictions (using :func:`cross_val_predict`) on which to
+        determine calibration parameters.
```

Comment:
May I suggest the following

> When ``ensemble`` is False, the cross validation generator is used to
  compute predictions (using :func:`cross_val_predict), and the union of these predictions is used for training the sigmoid or isotonic model. The  `base_estimator` is then fit on the whole data